### ISSUE TITLE
Add accessibility.descriptionFormat option

### ISSUE THREAD

**Issue by TorsteinHonsi** [highcharts/highcharts#14878]
#### Expected behaviour
For every formatter function there should be a string format.

#### Actual behaviour
`accessibility.descriptionFormatter` has no string format equivalent. This has two consequences: 1) In GUI's like Everviz, users are not able to use the option. 2) The options object is not JSON compatible.

#### Live demo with steps to reproduce
https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/highcharts/demo/x-range

With a format string, it would be something like this (except the index is 0-based, but we could probably work around that by either a custom option or a new property):

```js
    accessibility: {
        point: {
            /*
            descriptionFormatter: function (point) {
                var ix = point.index + 1,
                    category = point.yCategory,
                    from = new Date(point.x),
                    to = new Date(point.x2);
                return ix + '. ' + category + ', ' + from.toDateString() +
                    ' to ' + to.toDateString() + '.';
            }
            */
            descriptionFormat: '{index}. {yCategory}, {x:%e. %b %Y} to {x2:%e. %b %Y}'
        }
    },
```

#### Product version
v8.2.2

**Comment by TorsteinHonsi** [comment]
I think at least the demos should be updated with a string format, so that we don't have a formatter function right in between the otherwise JSON-friendly config. So in that case probably `valueDescriptionFormat` would be good.

We don't _have_ to have a string format option to match exactly the formatter, so if the `valueDescriptionFormat` suffices, I guess it's good.

**Comment by oysteinmoseng** [comment]
Will go through and update the demos. Another point is the [accessibility.series.descriptionFormatter](https://api.highcharts.com/highcharts/accessibility.series.descriptionFormatter) option, though, for the series. This one does not currently have a format string equivalent. It hasn't been requested yet, but perhaps we should add one for consistency?
